src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let clone = Object.assign({}, state);
+
+  return actions.map((action) => {
```

Comment:
Issue related to checklist item 4: Instead of returning the clone inside the map function, you should push the clone to an array after processing each action and return the array at the end of the function.